packages/settings-library/tests/test_twilio.py

```diff
@@ -12,10 +10,6 @@
 def test_twilio_settings_within_envdevel(
     mock_env_devel_environment: dict[str, str], monkeypatch: MonkeyPatch
 ):
-    # in .env-devel these are for the oment undefined
-    with pytest.raises(ValidationError):
-        TwilioSettings.create_from_envs()
```

Comment:
If you are removing this, please create a test without the fixture that automatically loads the `.env-devel` (I think it comes in via `mock_env_devel_environment`) and ensure it raises an error.

Reply:
ignore this